README.md

```diff
@@ -1,2 +1,128 @@
-# raincatcher
-Feedhenry Raincatcher Core Repository
+# RainCatcher
+Feedhenry RainCatcher Core Repository
+
+## About RainCatcher
+
+## Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
+
+Check your Node.JS version running `node -v`
+
+### Setup
+
+```bash
+git clone https://github.com/feedhenry-raincatcher/raincatcher-core.git
+cd raincatcher
+npm install -g lerna
+npm install
+lerna bootstrap
+```
+
+## Repository folder structure
+
+This repository contains many subpackages managed through [Lerna](https://lernajs.io/), and they're
+contained in the following directories:
+
+<dl>
+  <dt>client/</dt>
+  <dd>Front-end packages, from API clients to user interface modules</dd>
+
+  <dt>cloud/</dt>
+  <dd>Cloud app packages, intended to be run on an Node.JS environment</dd>
+
+  <dt>demo/</dt>
+  <dd>Full-fledged Demo applications, showcasing the usage of multiple modules</dd>
```

Comment:
Definitely not a grammar hero here, but I think that we can just say:
"Reference implementation for mobile, web and server side applications using proposed UI framework"